### DISCUSSION TITLE
Updating the webtop image to alpine linux 3.20?

### DISCUSSION THREAD

**Discussion by tspindler-cms** [Aterfax/DCS-World-Dedicated-Server-Docker#86] [Q&A] [answered]
Hello,

I'm testing out a new CGI/ATC solution named Standalone Radar Control. The server binary provides a GUI and needs wine-9. I've now manually updated the existing webtop docker image to Alpine Linux 3.20 and with this wine-9. This seems to work with DCS and the Standalone Radar Control server. Is there a way to rebase the existing Alpine Linux 3.19 to Alpine Linux 3.20 permanently in the setup of DCS-World-Dedicated-Server-Docker? Currently any call to docker-compose up -d will revert to Alpine Linux 3.19. Sorry for the newbieish question, some doc links might help as well.

**Comment by Aterfax** [comment, marked as answer]
>Is there a way to rebase the existing Alpine Linux 3.19 to Alpine Linux 3.20 permanently in the setup of DCS-World-Dedicated-Server-Docker?

Short answer: Yes, but I have no control over the upstream image so you would need to talk to the lscr.io maintainers to make such a request if they had not updated their image ([they already appear to be updated and now based off Alpine 3.20](https://github.com/linuxserver/docker-webtop/blob/aa03d4aaecc05403681cb424acfaea940b99ae35/Dockerfile#L1)). 

In effect this has already happened. You need only wait for the next automated build of the `dcs-world-dedicated-server` image on Saturday.

>Currently any call to docker-compose up -d will revert to Alpine Linux 3.19.

If you are building your own modified image you should tag it differently i.e. (do not build it as `aterfax/dcs-world-dedicated-server`). This will avoid pulling the existing image from dockerhub over the top of the one you modified. You should also change the `docker-compose.yml` to use your own tagged image.

**Comment by tspindler-cms** [comment]
Thanks for your answer Aterfax, I will wait for the weekend and see if 3.20 get used then. If that fails I will try out the own modified image, which would be a first for me.
